packages/client/src/components/Dropdown/index.scss

```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700,900');
```

Comment:
we don't need to import this here, we already have imported it once in the root `index.css` file (but its missing the font weights i think), you can copy this line there

just a heads up though, i think the final wireframes aren't using Roboto anymore, but we can figure that out later